config/jobs/kubernetes/sig-scalability/sig-scalability-experimental-periodic-jobs.yaml

```diff
@@ -357,3 +357,59 @@ periodics:
           - --test-cmd-name=ClusterLoaderV2
           - --timeout=120m
           - --use-logexporter
+
+  # Experiemntal job confirming that node killer works. Fork of ci-kubernetes-e2e-gci-gce-scalability.
+  # To be removed after Chaos Monkey flags are enabled in ci-kubernetes-e2e-gci-gce-scalability
+- interval: 30m
+  name: ci-kubernetes-e2e-gci-gce-scalability-node-killer
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+    preset-e2e-scalability-common: "true"
+  spec:
```

Comment:
You need to add `annotations` with testGrid settings

Reply:
Added. Thanks!